google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/project/ProjectSelectionDialog.java

```diff
@@ -211,11 +217,21 @@ public void onFailure(Throwable throwable) {
     }
   }
 
-  /** Preserves currently selected account and refreshes accounts and project lists. */
-  private void refreshDialog() {
+  /**
+   * Preserves currently selected account if requested and refreshes accounts and project lists.
+   *
+   * @param keepPreviousSelection true to keep current user selected after refresh. false to select
+   *     active user instead.
+   */
+  private void refreshDialog(boolean keepPreviousSelection) {
```

Comment:
would it be possible to directly pass in the user to select rather than having a flag like this?

Reply:
good idea, let me rework it that way,